metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -153,6 +154,7 @@ def run(
     max_parallelism=None,
     workflow_timeout=None,
     wait_for_completion=False,
+    **pipeline_parameters
```

Comment:
Why is it `**pipeline_parameters` in line 157 in `kfp_cli.py` and `pipeline_parameters=None` in line 7 of `kfp_step_function.py`?

Would it be possible to place an inline comment on what data type `pipeline_parameters` is, because it seems to switch between a Python dict and a json string in different parts of the code.

Reply:
Fixed by renaming serialized param name to `*_json`. There is possibility to get rid of json serialization in up coming changes.